examples/linear_deformation.py

```diff
@@ -0,0 +1,1187 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Example of shape-based image reconstruction using linearized deformations.
+"""
+
+# Imports for common Python 2/3 codebase
+from __future__ import print_function, division, absolute_import
+from future import standard_library
+from builtins import super
+from numbers import Number
+from odl.operator.operator import Operator, OperatorComp
+import odl
+import numpy as np
+import time
+import matplotlib.pyplot as plt
+standard_library.install_aliases()
+
+
+class Functional(Operator):
```

Comment:
As the docstring says, the code from [here](https://github.com/odlgroup/odl/pull/448/files#diff-5c26f74ebc86d7b067454b347a224c50R35) to [there](https://github.com/odlgroup/odl/pull/448/files#diff-5c26f74ebc86d7b067454b347a224c50R203) was a quick hack to get some notion of functional going. This is now taken care of in [a branch of its own](https://github.com/odlgroup/odl/tree/issue-246__functional).

That means that you should remove this code. See summary below on what to do.